client/sclient/main.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
This package doesn't define a `main` function, may I suggest:

```suggestion
package client
```